### ISSUE TITLE
[Accessibility issue] Text not visible in DevTools under the KeyNav section

### ISSUE THREAD

**Issue by pratyaksh1610** [musescore/MuseScore#16646]
### Issue type

UI bug

### Bug description

`Last clicked..` text not visible in DevTools under the KeyNav section in dark mode. It is an accessibility issue which shall be taken into account.

### Steps to reproduce

Go to DevTools -> KeyNav

### Screenshots/Screen recordings

![last](https://user-images.githubusercontent.com/76740999/222714562-bc05ff66-2c43-428e-a6ab-e5ed18468477.png)

### MuseScore Version

4.1.0

### Regression

I don't know

### Operating system

Windows 10
